101-app-service-create-managed-certificate-apex/README.md

```diff
@@ -0,0 +1,43 @@
+# Create a Managed Certificate (Free) for an APEX domain (aka root/naked domain)
+
+[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-app-service-create-managed-certificate-apex%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-app-service-create-managed-certificate-apex%2Fazuredeploy.json)
+
+
+To deploy this template, you need to have the following resources:
+
+1. The App Service Plan (serverFarm) resource name that the Web App is running.
```

Comment:
add a prereq template for this

Reply:
I added, but let me know how to add the A record, if we go to manual review